composer.json

```diff
@@ -35,11 +40,6 @@
         "psr/http-client-implementation": "1.0",
         "php-http/client-implementation": "1.0"
     },
-    "scripts": {
-        "test": "vendor/bin/phpunit",
-        "test-ci": "vendor/bin/phpunit --coverage-clover build/coverage.xml"
```

Comment:
please do not remove these, we have them with all php-http adapters and `composer test` is also mentioned in the readme.

Reply:
scripts section is below, this section just have been here duplicated somehow

Reply:
oh, did not see that. yeah we only need it once :-/  thanks for fixing.